var/spack/repos/builtin/packages/py-tensorflow/package.py

```diff
@@ -30,6 +30,8 @@ class PyTensorflow(Package, CudaPackage):
     maintainers = ["adamjstewart", "aweits"]
     import_modules = ["tensorflow"]
 
+    version("2.7.4-rocm-enhanced", sha256="45b79c125edfdc008274f1b150d8b5a53b3ff4713fd1ad1ff4738f515aad8191",
+            url = "https://github.com/ROCmSoftwarePlatform/tensorflow-upstream/archive/refs/tags/v2.7.0.tar.gz")
```

Comment:
Can you sort these versions numerically?

Reply:
will do.. yes.. for other versions, i can definitely add the rocm support but will need to talk to TF team internally and get them tested..